pkg/GAPJulia/JuliaInterface/julia/gap.jl

```diff
@@ -44,6 +44,9 @@ Base.setproperty!(x::MPtr, f::Union{AbstractString,Int64}, v) = Globals.ASS_REC(
 Base.zero(x::GAPInputType_internal) = Globals.ZERO(x)
 Base.one(x::GAPInputType_internal) = Globals.ONE(x)
 Base.:-(x::GAPInputType_internal) = Globals.AINV(x)
+Base.:-(x::Int64,y::GAP.MPtr) = GAP.julia_to_gap(x)-y
```

Comment:
I am actually puzzled this is necessary. Arithmetic between `Int64` and `MPtr` should already be defined. If not, we should add it there, not as a special case.
Moreover, `julia_to_gap` is a no-op on all integers smaller than 2^60, so are you sure this is necessary?

Reply:
https://github.com/oscar-system/GAP.jl/blob/master/pkg/GAPJulia/JuliaInterface/julia/gap.jl#L51 This line plus the `-` below in the loop should actually be sufficient to implement this function. Do you call it on integers larger than 2^60?

Reply:
My code seems to be working without this now, I don't remember why I needed this conversion.